include/engine/guidance/step_maneuver.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef ENGINE_GUIDANCE_STEP_MANEUVER_HPP
+#define ENGINE_GUIDANCE_STEP_MANEUVER_HPP
+
+#include "util/coordinate.hpp"
+#include "extractor/turn_instructions.hpp"
+
+namespace osrm
+{
+namespace engine
+{
+namespace guidance
+{
+
+struct StepManeuver
+{
+    util::FixedPointCoordinate location;
+    double heading_before;
+    double heading_after;
+    extractor::TurnInstruction instruction;
+};
```

Comment:
Oh this is captured by the `RouteStep` by the `way_name`. Sorry should document the relation better here. `StepManeuver` is the maneuver used to navigate on the given `RouteStep`